modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1082,6 +1082,8 @@ protected function render_tab_titles_html( $item_settings ): string {
 			$css_classes[] = 'elementor-animation-' . $item_settings['settings']['hover_animation'];
 		}
 
+		$aria_label = esc_html__( '. Open links with Enter or Space, close with Escape, navigate with Arrow Keys', 'elementor' );
```

Comment:
You should not start a translation string with a period. It's had to translate this type of string.

Developers, that see the code, understand the logic. But translators don't.

Reply:
Updated.